### ISSUE TITLE
LayerNormalization wrong "axis" parameter coded

### ISSUE THREAD

**Issue by CarlosNacher** [keras-team/tf-keras#173]
**System information**.
- Have I written custom code (as opposed to using a stock example script provided in Keras): YES
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): 
- TensorFlow installed from (source or binary): I don't know. In Colab
- TensorFlow version (use command below): v2.12.0-rc1-12-g0db597d0d75 2.12.0
- Python version: 3.10.12
- GPU model and memory: None

**Describe the problem**.

Layer Normalization with axis = -1 normalises calculating the mean of each pixel across the channels, instead of the mean of all pixels in a example across the channels (HxW). If you set axis=-1, then there are as many parameters as channels. But if you test with a generated array, you see that it does not calculate n_channels averages, but actually calculates an average for each pixel (HxW).

**Describe the current behavior**.

Above

**Describe the expected behavior**.

I don't find the sense of having n_channels parameters if when you test LN, you see that there are HxW means calculated. I've put all this in a notebook. See below (is more explained).

**Standalone code to reproduce the issue**.

In this Notebook I do some tests: https://colab.research.google.com/drive/1EMXLaUkCLO-J_svFOJ_fMkMCoUFQ4Ewu?usp=sharing
